src/compiler/checker.ts

```diff
@@ -12981,6 +12981,20 @@ namespace ts {
                 return result;
             }
 
+            function propegateSidebandVarianceFlags(typeArguments: readonly Type[], variances: VarianceFlags[]) {
+                if (outofbandVarianceMarkerHandler) {

```

Comment:
In the two places you call this function you've already checked that this is truthy

Reply:
Fair enough 🤷‍♀